modules/admin/admin-ui/src/main/resources/web/admin/common/js/liveedit/LiveEditModel.ts

```diff
@@ -80,13 +80,11 @@ module api.liveedit {
                 hasDefaultPageTemplate = this.pageModel.hasDefaultPageTemplate(),
                 hasApplications = this.siteModel.getApplicationKeys().length > 0;
 
-            var isRenderable = hasApplications || hasController || hasDefaultPageTemplate,
-                isRenderableSite = isSite && isRenderable,
-                isRenderableTemplate = isTemplate && hasController;
-            
-            return isRenderableSite || isRenderableTemplate || hasDefaultPageTemplate;
+            var isRenderable = hasApplications || hasController || hasDefaultPageTemplate;
+
+            return isRenderable && (isTemplate || isSite);
```

Comment:
With this condition the page editor will not render preview for a page with a template. For example, open any content under Superhero/posts folder.